Reviews/337 -  Royal web wallet and casper connector/1- Development/README.md

```diff
@@ -0,0 +1,394 @@
+
+# Code / Unit / Automated Testing
+
+## Code
+
+The reviewer was not able to successfully run the test scenario provided in the README file.
+There are no documentation on unit or automated tests.
+However, the reviewer run the command "yarn test" to see if there are somes tests. 
+
+```bash
+
+[reviewer@rocky8-mae-vm Draw-Bridge]$ sudo yarn test
+yarn run v1.22.18
+$ yarn workspace @scaffold-eth/hardhat test
+$ hardhat test --network hardhat
+
+  My Dapp
+    YourContract
+      ✓ Should deploy YourContract
+      setPurpose()
+0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 set purpose to Test Purpose
+        ✓ Should be able to set a new purpose
+
+·-------------------------------|---------------------------|-------------|-----------------------------·
+|      Solc version: 0.8.4      ·  Optimizer enabled: true  ·  Runs: 200  ·  Block limit: 30000000 gas  │
+································|···························|·············|······························
+|  Methods                                                                                              │
+·················|··············|·············|·············|·············|···············|··············
+|  Contract      ·  Method      ·  Min        ·  Max        ·  Avg        ·  # calls      ·  usd (avg)  │
+·················|··············|·············|·············|·············|···············|··············
+|  YourContract  ·  setPurpose  ·          -  ·          -  ·      34210  ·            1  ·          -  │
+·················|··············|·············|·············|·············|···············|··············
+|  Deployments                  ·                                         ·  % of limit   ·             │
+································|·············|·············|·············|···············|··············
+|  YourContract                 ·          -  ·          -  ·    1074922  ·        3.6 %  ·          -  │
+·-------------------------------|-------------|-------------|-------------|---------------|-------------·
+
+  2 passing (4s)
+
+Done in 15.29s.
+
+```
+
+## Unit / Automated Testing
+
+The project contains only 2 unit tests. These tests run successfully. There are no negative path tests at all.
+
+Requirement | Finding
+------------ | -------------
+Unit Tests - At least one positive path test | Pass with notes
+Unit Tests - At least one negative path test | Failed
+Unit Tests - Additional path tests | Failed
+
+# Documentation
+
+### Code Documentation
+
+The code is overall well documented, but the reviewer thinks that some improvements are needed  :
+
+**- Unused code can be removed in order to improve code readability (eg : hardhat/contracts/00_deploy_your_contract.js).** 
+
+**- Secret informations (like API KEY) should not be committed to github (some "api key" found hardcoded in the repository (eg : Alchemy))**
+  
+Requirement | Finding
+------------ | -------------
+Code Documented | PASS with note
```

Comment:
As per the CRDAO template for the DxD submissions, it needs to be `PASS`, `FAIL` or `PASS with Notes`. Attention to detail is appreciated.